### PR TITLE
Initial work to get saving of JSON for End of Article working

### PR DESCRIPTION
**Pull request by danbowles**

### CODE REVIEW

**Review by jomurgel — APPROVED** [review]
Adorable 🍞

File: includes/apple-exporter/components/class-end-of-article.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * @subpackage Apple_Exporter\Components
+ */
+
+namespace Apple_Exporter\Components;
+
+/**
+ * Represents an Article Format End_Of_Article. It gets generated automatically
+ * so not much to do here but define the static JSON.
+ *
+ * @since 0.4.0
```
Comment (kevinfodness) [inline includes/apple-exporter/components/class-end-of-article.php]: This should be `2.1.0`

File: includes/apple-exporter/components/class-end-of-article.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ *
+ * @package Apple_News
+ * @subpackage Apple_Exporter\Components
+ */
+
+namespace Apple_Exporter\Components;
+
+/**
+ * Represents an Article Format End_Of_Article. It gets generated automatically
+ * so not much to do here but define the static JSON.
```
Comment (kevinfodness) [inline includes/apple-exporter/components/class-end-of-article.php]:
```suggestion
 * Represents an End of Article module. The module initializes as empty and
 * is defined by the user via the Customize JSON feature. If non-empty, it is
 * inserted onto the end of the article body.
```

**Review by kevinfodness — APPROVED** [review]
Two minor tweaks to the comments - can be done now or as part of the next PR. 🍣

**Pull request merged**